utility/WiFiServerStream.h

```diff
@@ -70,14 +70,15 @@ class WiFiServerStream : public WiFiStream
     if( connect_client() ) return true;
 
     stop();
-    
+
     if( !_listening && WiFi.status() == WL_CONNECTED )
     {
       // start TCP server after first WiFi connect
+      _server = WiFiServer(_port);
```

Comment:
It turns out that this change was not necessary (I check to be sure). The real issue was that the `_client` reference was never set.